projects/ngrx.io/src/app/custom-elements/contributor/contributor-list.component.ts

```diff
@@ -13,34 +15,58 @@ import { LocationService } from 'app/shared/location.service';
        (click)="selectGroup(name)">{{name}}</a>
   </div>
   <section *ngIf="selectedGroup" class="grid-fluid">
+    <div *ngIf="selectedGroup.name === 'Community'">
+      <mat-paginator
+        [length]="350"
+        [pageIndex]="contributorPage$ | async"
```

Comment:
Just curious. The `ngrxPush` pipe could be used here, right? Taking the size of this application into consideration, would the pipe make difference here?

Reply:
Yes, it could be, but it wouldn't make much of a difference. The main thing is that I could use the observable without an ngIf + variable assignment.